cmd/occollector/app/builder/builder.go

```diff
@@ -102,6 +103,9 @@ func (cfg *JaegerReceiverCfg) InitFromViper(v *viper.Viper) (*JaegerReceiverCfg,
 type OpenCensusReceiverCfg struct {
 	// Port is the port that the receiver will use
 	Port int `mapstructure:"port"`
+
+	// TLSCredentials is a (cert_file, key_file) configuration.
+	TLSCredentials *config.TLSCredentials `mapstructure:"tls_credentials"`
```

Comment:
note: since we postponed that PR moving all underscores to dashes I have to notice that the collector only uses dashes... let's try to get to the config resolution this week...